pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -12,6 +12,11 @@ WARNING: DO NOT edit .pxi FILE directly, .pxi is generated from .pxi.in
 from pandas._libs.tslibs.util cimport get_c_string
 from pandas._libs.missing cimport C_NA
 
+cdef extern from "Python.h":
+    # Note: importing extern-style allows us to declare these as nogil
```

Comment:
This is interacting with Python objects so it needs to be called with the GIL

Reply:
Will remove it, I am quite new to cython, so I have to read up a bit on these things.

Reply:
If you are getting warnings / errors when trying to cythonize you might need a `with gil:` block in your error handler

Reply:
I am not getting any cython errors, so I am probably good.